src/StreetNameRegistry.Producer.Ldes/ProducerProjections.cs

```diff
@@ -0,0 +1,394 @@
+                                break;
+                            case Language.French:
+                                streetNameDetail.HomonymAdditionFrench = null;
+                                break;
+                            case Language.German:
+                                streetNameDetail.HomonymAdditionGerman = null;
+                                break;
+                            case Language.English:
+                                streetNameDetail.HomonymAdditionEnglish = null;
+                                break;
+                            default:
+                                throw new ArgumentOutOfRangeException(nameof(language), language, null);
+                        }
+                    }
+
+                    UpdateVersionTimestamp(streetNameDetail, message.Message.Provenance.Timestamp);
+                }, ct);
+            });
+
+            When<Envelope<StreetNameWasRemovedV2>>(async (context, message, ct) =>
+            {
+                await context.FindAndUpdateStreetNameDetail(message.Message.PersistentLocalId, streetNameDetail =>
+                {
+                    streetNameDetail.IsRemoved = true;
+                    UpdateVersionTimestamp(streetNameDetail, message.Message.Provenance.Timestamp);
+                }, ct);
+            });
+
+            When<Envelope<MunicipalityNisCodeWasChanged>>((context, message, _) =>
+            {
+                var streetNamesToChange = context
+                    .StreetNames
+                    .Local
+                    .Where(s =>
+                        s.MunicipalityId == message.Message.MunicipalityId)
+                    .Union(context.StreetNames.Where(s =>
+                        s.MunicipalityId == message.Message.MunicipalityId));
+
+                foreach (var streetNameDetail in streetNamesToChange)
+                    streetNameDetail.NisCode = message.Message.NisCode;
+
+                return Task.CompletedTask;
+            });
+
+            When<Envelope<MunicipalityBecameCurrent>>(DoNothing);
+            When<Envelope<MunicipalityFacilityLanguageWasAdded>>(DoNothing);
+            When<Envelope<MunicipalityFacilityLanguageWasRemoved>>(DoNothing);
+            When<Envelope<MunicipalityOfficialLanguageWasAdded>>(DoNothing);
+            When<Envelope<MunicipalityOfficialLanguageWasRemoved>>(DoNothing);
+            When<Envelope<MunicipalityWasCorrectedToCurrent>>(DoNothing);
+            When<Envelope<MunicipalityWasCorrectedToRetired>>(DoNothing);
+            When<Envelope<MunicipalityWasImported>>(DoNothing);
+            When<Envelope<MunicipalityWasMerged>>(DoNothing);
+            When<Envelope<MunicipalityWasNamed>>(DoNothing);
+            When<Envelope<MunicipalityWasRetired>>(DoNothing);
+            When<Envelope<StreetNameHomonymAdditionsWereCorrected>>(DoNothing);
+            When<Envelope<StreetNameHomonymAdditionsWereRemoved>>(DoNothing);
```

Comment:
They are implemented

Reply:
Indeed, then this is also incorrect in `StreetNameDetailV2Projections`. There's also 2 handlers for those 2 events it seems, I'll remove the `DoNothing` there as well.

Reply:
You could even argue that you shouldn't be able to have multiple handlers for the same event within the same projection.

Reply:
done